Fix index decoding in aux channels

aacuevas opened the pull request:
Fixed issue that was causing aux channels become mangled

As a question, in here:
https://github.com/open-ephys-plugins/acquisition-board/blob/d71507ad0107d5ee2442481b103b8a3671a693cc/Source/devices/oni/AcqBoardONI.cpp#L1777

Should be use this monolithic counter, or the Rhythm sample counter?

aacuevas committed:
Fix index decoding in aux channels

jsiegle commented:
Rhythm sample counter may be better, but we should test that more extensively before making that change.

Can you also bump the plugin version number to 1.1.3?

bparks13 committed:
Reduce severity of message when no Opal Kelly board is found

bparks13 committed:
Bump version to 1.1.3

bparks13 commented:
I bumped the version number so that the actions pass.

I also took the opportunity to change the log from error to console when no Opal Kelly board is found.

Question; there is an outstanding issue we've discussed which is to give a pop-up indicating if the board firmware version is too old. What is the cutoff for that version? Is it any versions 0.x that need to be updated? That might be a good addition to add in to this patch version.

bparks13 reviewed (approved):
Tested this with one 64 channel headstage, and it correctly streams AUX data.

jsiegle commented:
@aacuevas what is the minimum gateware version needed for this plugin? It looks like 1.5.3 is the latest. Should we just say 1.5.x to be on the safe side?

bparks13 committed:
Add popup to update gateware if major version is reported as 0

- This will also catch version 1.0 of the gateware, as it reports the major version as 0 and the minor version as 1, but this is okay as users should still update to a higher version to make use of all the available features

anjaldoshi committed:
Change hyperlink text color from purple to deepskyblue

aacuevas commented:
> Rhythm sample counter may be better, but we should test that more extensively before making that change.
> 
> Can you also bump the plugin version number to 1.1.3?

@jsiegle sorry, my email seems to have stopped working yesterday and I got all the notifications today.

Rhythm sample counter is what we used to use before the advent of real time timestamps, so I imagine they should be safe. But it does not hurt to check just in case.
